CONTRIBUTING.md

```diff
@@ -1,8 +1,109 @@
+
+A TSC meeting should be announced via the nnstreamer-announce LFAI mailing list (the mailing list). Other media (GitHub issues, gitter.im, social media, LFAI event calendar, and so on) may also be used along with the mailing list.
+
+A TSC meeting should be held at a publically accessible place. A TSC meeting is, by default, held conventionally (offline meeting) in a place where the chairperson or its deputy has announced. However, alternatively, a TSC meeting may be held virtually (audio or video conference) with publically available media that are declared with the meeting announcement. A conventionally-held TSC meeting may include audio or video conferences to help those who cannot present physically.
+
+A TSC meeting is, by default, recorded or scripted for the general public. The URLs or the contents of the recordings or scripts should be available via the mailing list. Alternatively, a live video stream may be broadcasted via methods declared by the mailing list.
+
+In the case of the unavailability of the mailing list, a pinned GitHub issue of https://github.com/nnstreamer/nnstreamer can be used instead.
+
+
+## Unavailability of a Chairperson or its Deputy
+A chairperson should assign its deputy or hold a TSC meeting to elect a successor when he/she resigns.
+
+If there is no chairperson or its deputy available, the most senior and available committer should announce and hold a TSC meeting to elect a chairperson. An available committer is a committer who has been submitting code commits or code reviews to the main project (nnstreamer.git) during the last 30 days. A senior committer is a committer who has committed codes to the main GitHub project (nnstreamer.git) before other available committers. Unavailability of chairman and deputy can be declared via the mailing-list by a committer. The unavailability declaration becomes effective after the declaring committer tries to contact the chairman or deputy personally, the declaration is acknowledged by other two or more committers, and there are no responses from chairman or deputy via the mailing list or GitHub issues of the main project (nnstreamer.git) within 30 days. With the effective unavailability declaration, the available senior committer becomes the deputy who holds the TSC meeting to elect a chairperson within the next 30 days. If the available senior committer fails to do this within the given 30 days after the effective declaration, the Linux Foundation Series manager may designate any TSC voting member of committer as a deputy chairperson and any TSC voting member or committer may contact Linux Foundation to initiate this process.
```

Comment:
Q. I am having hard time to catch the meaning of  `A senior committer`. Would the definition include a person who hasn't been committed for a long time but committed `before other available committers`?

Reply:
Do you suggest that "the most senior among available committers" will be more clear than "the most senior and available committer"?

Reply:
I think that is more clear. Thank you for the clarification 👍

Reply:
Updated!